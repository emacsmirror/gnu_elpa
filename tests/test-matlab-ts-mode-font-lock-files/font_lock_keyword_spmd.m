% -*- matlab-ts -*-
% https://github.com/acristoffers/tree-sitter-matlab/issues/25
parpool(3)
spmd
    q = magic(spmdIndex + 2);
end
