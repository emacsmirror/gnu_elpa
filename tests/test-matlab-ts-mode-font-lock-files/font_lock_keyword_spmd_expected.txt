c ccc ccccccccc ccc
c cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc
BBBBBBBbnb
kkkk
    v o BBBBBbBBBBBBBBB o nbD
kkk
